az_quickstart.install

```diff
@@ -36,6 +36,15 @@ function az_quickstart_install() {
   ]);
   $shortcut->save();
 
+  // Allow editors to access icon library.
+  $shortcut = Shortcut::create([
+    'shortcut_set' => 'default',
+    'title' => t('Icon library'),
+    'weight' => -18,
+    'link' => ['uri' => 'internal:/admin/appearance/ui/icons'],
+  ]);
+  $shortcut->save();
```

Comment:
Could we move this shortcut creation code into the az_icons module (in a `hook_install()` implementation)?  I believe that would also make the DB update not necessary.